join.spec.md

```diff
@@ -20,20 +20,21 @@ graph LR
   style core fill:none,stroke:fuchsia,color:fuchsia;
 ```
 
-This document defines schema elements for describing [core schemas](/core/v0.1) which **join** multiple **subgraph** schemas into a single **supergraph** schema.
+This document defines a [core feature](https://specs.apollo.dev/core/v0.1) named `join` for describing [core schemas](https://specs.apollo.dev/core/v0.1) which **join** multiple **subgraph** schemas into a single **supergraph** schema.
 
 This specification provides machinery to:
 - define [subgraphs](#def-subgraph) with the {join__Graph} enum and the {@join__graph} directive
 - assign fields to subgraphs with {@join__field}
-- declare additional data required and provided by subgraph field resolvers with the [`requires`](#@join__field/requires) and [`provides`](#@join__field/provides) arguments
+- declare additional data required and provided by subgraph field resolvers with the `requires` and `provides` arguments to {@join__field}
+- assign keys and ownership to types with {@join__type} and {@join__owner}
```

Comment:
This bullet feels like it needs more context (the why behind keys and ownership) but none of the previous bullets do that. However, most of the previous bullets feel a bit more self-explanatory?

I haven't made up my mind on this suggestion, mostly just a thought worth considering.

Reply:
I think this section needs to be pretty succinct, but I added a link in 1d7d243 which maybe will help?

Reply:
Yeah that works for me